app/(white_navbar)/explore/RetroPGF4/_component/Tab/BadgeholderTab.tsx

```diff
@@ -0,0 +1,253 @@
+import { FC, Fragment, Suspense, useEffect, useMemo, useState } from 'react'
+import BadgeholderCard from './component/BadgeholderCard'
+import { BadgeholderMetrics } from '../../RetroType4'
+import Image from 'next/image'
+import InputAndFilterBadgeholderTab from './component/InputAndFilterBadgeholderTab'
+import {
+  CheckBoxStateBadgeholderType,
+  ExploreRoundState,
+} from '../ExploreRoundType'
+import { isLetter, shuffle } from '@/app/lib/utils'
+import CheckBoxFilterBadgeholder from './component/CheckBoxFilterBadgeholder'
+import { Pagination } from 'react-headless-pagination'
+import { itemsPerPage } from '../Text'
+
+interface BadgeholderTabProps {
+  badgeholderData: BadgeholderMetrics[]
+}
+
+const BadgeholderTab: FC<BadgeholderTabProps> = ({ badgeholderData }) => {
+  const [currentPage, setCurrentPage] = useState<number>(0)
+  const [loading, setLoading] = useState(() => true)
+  const [search, setSearch] = useState<string>(() => '')
```

Comment:
in line 21 and 22 why set the default value to `()=>true` and `()=>''` instead of just `true` or `''`

Reply:
This is what is rabbit suggest, it's to that it's avoid something I can't remember but I think go with normal one is okay.